data/build.gradle.kts

```
@@ -0,0 +1,12 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
```

```suggestion
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8
```
1.8 버전을 사용하시는 것을 권장해요 (이하 관련 내용 동일)